spec/lib/whitehall_importer/change_history_spec.rb

```diff
@@ -0,0 +1,81 @@
+RSpec.describe WhitehallImporter::ChangeHistory do
```

Comment:
Looks like we're missing a `describe "#for"` block

Reply:
Sure... added